pkg/policy/api/rule.go

```diff
@@ -178,6 +180,13 @@ type EgressRule struct {
 	//
 	// +optional
 	ToCIDR []CIDR `json:"toCIDR,omitempty"`
+
+	// ToEntities is a list of special entities which the endpoint subject
+	// to the rule is allowed to initiate connections to. Supported entities are
```

Comment:
... to which the endpoint subject to the rule is allowed to initiate connections.

Reply:
will do